### PR TITLE
Grid layout

### PR DESCRIPTION
**Pull request by DavidRojas1612**
## Summary
- Layout html using grid css.

## Screenshots
image

image

## Trello Card
https://trello.com/c/EGE9IROa

### CODE REVIEW

File: maquetado/scss/styles.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .section-project {
-  display: flex;
-  flex-direction: column;
+  display: grid;
+  grid-gap: 5px;
```
Comment (CarlosHWolox) [inline maquetado/scss/styles.scss]: the `react` is too close to the border line, maybe adding a greater value to this gap could make it look better?

File: maquetado/scss/tech.scss [hunk 2]
```diff
@@ -8,28 +8,30 @@
...
-    left: -10px;
-    position: absolute;
-    top: -10px;
-    transition: all 0.30s cubic-bezier(0.785, 0.135, 0.15, 0.86);
-    width: 100%;
+  &s-container {
+    display: grid;
+    grid-gap: 5px;
+    grid-template-areas: 'first-tech second-tech third-tech' 'first-tech quarter-tech fifth-tech';
+    grid-template-columns: repeat(3, 300px);
```
Comment (CarlosHWolox) [inline maquetado/scss/tech.scss]: could add that 300px as a variable

File: maquetado/scss/tech.scss [hunk 3]
```diff
@@ -8,28 +8,30 @@
...
-    height: 100%;
-    left: -10px;
-    position: absolute;
-    top: -10px;
-    transition: all 0.30s cubic-bezier(0.785, 0.135, 0.15, 0.86);
-    width: 100%;
+  &s-container {
+    display: grid;
+    grid-gap: 5px;
+    grid-template-areas: 'first-tech second-tech third-tech' 'first-tech quarter-tech fifth-tech';
```
Comment (CarlosHWolox) [inline maquetado/scss/tech.scss]: this is not necessary, you could use a selector of `first-child`

File: maquetado/scss/tech.scss [hunk 4]
```diff
@@ -8,28 +8,30 @@
...
+    justify-self: center;
   }
 
-  &:hover::before {
-    left: 10px;
-    top: 10px;
+  &:first-of-type {
+    grid-area: first-tech;
+
+    &::before {
```
Comment (CarlosHWolox) [inline maquetado/scss/tech.scss]: i think this is not necessary, you could use a `grid-gap`

File: maquetado/scss/tech.scss [hunk 5]
```diff
@@ -8,28 +8,22 @@
 .tech {
   @extend %equal-align;
 
-  background-color: $green;
   border-radius: 5px;
```
Comment (alejobh) [inline maquetado/scss/tech.scss]: this is not necessary and its making the green line looks with radius in the edges

Comment (alejobh) [reply maquetado/scss/tech.scss]: in fact, you dont need `position:relative` and `margin` :) check the new design and just use the padding and the grid gap properties

**Pull request merged**